docs/run.md

```diff
@@ -128,12 +128,23 @@ https://docs.microsoft.com/en-us/azure/virtual-machines/linux/cli-ps-findimage):
 
 4. Run LISA
 
-    Use above `<subscription id>` and `<private key file>` to replace in below command. It may take several minutes to complete.
+    Use above `<subscription id>` and `<private key file>` to replace
+    in commands below. They may take several minutes to complete,
+    depending on the complexity of the passed runbooks. A Marketplace
+    image *must* be provided (by its URN).
 
     ```bash
-    lisa -r ./microsoft/runbook/azure.yml -v subscription_id:<subscription id> -v "admin_private_key_file:<private key file>"
+    lisa -r ./microsoft/runbook/azure.yml -v subscription_id:<subscription id> -v "admin_private_key_file:<private key file>" -v "marketplace_image:Debian:debian-10:10:latest"
```

Comment:
In this document, it makes sure the environment is correct with minimum arguments. I have a task to make private key as optional. After that, only subscription is required. That would be much easier to have a first try.
I understand in most scenarios, the image should be specified. So we can add a notes below the command to explain this parameter can specify a marketplace image.
Feel free to create a task to me. I will fix this document later.

Reply:
```
2021-04-19 18:55:08.033 INFO LISA completed in 1.086 sec
2021-04-19 18:55:08.034 ERROR LISA cannot find variable 'marketplace_image', make sure its value filled in runbook, command line or environment variables.
Traceback (most recent call last):

```
without the market_place arg. I'll open an issue for that, then